modules/users/client/components/ProfileOverview.component.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * A panel with basic user info in user profile.
+ * It wraps Avatar, a Modal with Avatar and ProfileViewBasics components.
+ * @param {Object} profile - displayed user's profile data
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Avatar from './Avatar.component';
+import ProfileViewBasics from './ProfileViewBasics';
+import { Modal } from 'react-bootstrap';
+
+export default function ProfileOverview({ profile }) {
+  const [isAvatarModalOpen, setIsAvatarModalOpen] = useState(false);
+
+  return (<>
+    {/* panel with avatar and basic info */}
+    <div className="panel panel-default profile-overview">
+      {/* avatar */}
+      <a
+        className="hidden-xs"
+        aria-hidden={true}
+        onClick={() => setIsAvatarModalOpen(true)}
+      >
+        <Avatar
+          user={profile}
+          size={256}
+          link={false}
+        />
+      </a>
+
+      {/* basic info panel */}
+      <div className="panel-body">
+        <div className="profile-sidebar-section">
+          <ProfileViewBasics profile={profile} />
+        </div>
+      </div>
+    </div>
+
+    {/* modal with avatar */}
+    <Modal
+      show={isAvatarModalOpen}
+      onHide={() => setIsAvatarModalOpen(false)}
+      className="modal-avatar"
+    >
+      <Avatar
```

Comment:
Now when image is opened, it cannot be closed by re-clicking. It can only be closed by clicking at the area outside the image:

<img width="887" alt="Screenshot 2019-11-10 at 20 51 22" src="https://user-images.githubusercontent.com/87168/68549006-13613100-03fc-11ea-8cd1-5bbc2b94f407.png">

Would just adding `onClick={handleClose}` in `Avatar` work? Or wrapping it in `<button>`?

Reply:
Good catch! :+1:
fixed with  68f5a0e